packages/valory/agents/oracle/aea-config.yaml

```diff
@@ -89,17 +89,17 @@ public_id: valory/p2p_libp2p_client:0.1.0
 type: connection
 config:
   nodes:
-  - uri: ${P2P_URI:str:localhost:11000}
-    public_key: ${P2P_PUBLIC_KEY:str:03c74dbfbe7bbc1b42429f78778017a3cd7eaf9d59d1634c9505a3f7c1a9350e71}
+  - uri: ${str:localhost:11000}
+    public_key: ${str:03c74dbfbe7bbc1b42429f78778017a3cd7eaf9d59d1634c9505a3f7c1a9350e71}
 cert_requests:
-- identifier: acn
-  ledger_id: ethereum
-  message_format: '{public_key}'
-  not_after: '2023-01-01'
-  not_before: '2022-01-01'
-  public_key: ${P2P_PUBLIC_KEY:str:03c74dbfbe7bbc1b42429f78778017a3cd7eaf9d59d1634c9505a3f7c1a9350e71}
-  save_path: .certs/acn_cosmos_11000.txt
-is_abstract: false
+- identifier: ${str:acn}
+  ledger_id: ${str:ethereum}
+  message_format: ${str:'{public_key}'}
+  not_after: ${str:'2023-01-01'}
+  not_before: ${str:'2022-01-01'}
+  public_key: ${str:03c74dbfbe7bbc1b42429f78778017a3cd7eaf9d59d1634c9505a3f7c1a9350e71}
+  save_path: ${str:.certs/acn_cosmos_11000.txt}
+is_abstract: ${bool:false}
```

Comment:
Cannot successfully run `autonomy hash all` after this change. Getting the following error:

```
ValueError: Configuration of component (connection, valory/p2p_libp2p_client:0.1.0) is not valid. sequence item 1: expected str instance, int found
```

Reply:
`is_abstract` is not overridable and AFAIC the only thing overridable in the `cert_request` object is a public key